app/lib/ui/components/buttons/component_button_customization.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
```

Comment:
rename the file removing _component_ to_ have  _button_customization.dart_

Reply:
done